### ISSUE TITLE
Manual Instrumentation in Java is missing dependencies and imports and this makes it a time waster

### ISSUE THREAD

**Issue by odupuy** [open-telemetry/opentelemetry.io#2029]
On https://opentelemetry.io/docs/instrumentation/java/manual The "set up" code below (first and last line) requires a lot of top level dependencies.
```
Resource resource = Resource.getDefault()
...
  .buildAndRegisterGlobal();
```
While you can find them it is a lengthy and very painful process.
Please provide the corresponding maven "top level" dependencies below (verified with `mvn dependency:analyze`). Use a collapsible section if you want to keep the things short and sweet if this was the goal. Note that `opentelemetry-semconv` does not exist (yet) as `1.20.1` but only as `1.20.1-alpha`
```
</project>
    <dependencyManagement>
        <dependencies>
            <dependency>
                <groupId>io.opentelemetry</groupId>
                <artifactId>opentelemetry-bom</artifactId>
                <version>1.20.1</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>

    <dependencies>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-api</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-sdk</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-sdk-trace</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-sdk-metrics</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-sdk-common</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-exporter-otlp</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-context</artifactId>
        </dependency>
        <dependency>
            <groupId>io.opentelemetry</groupId>
            <artifactId>opentelemetry-semconv</artifactId>
            <version>1.20.1-alpha</version>
        </dependency>
    </dependencies>
</project>
```
And add then as well the imports for the setup code (collapsible if needed)
```
import io.opentelemetry.api.OpenTelemetry;
import io.opentelemetry.api.common.Attributes;
import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
import io.opentelemetry.context.propagation.ContextPropagators;
import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
import io.opentelemetry.sdk.OpenTelemetrySdk;
import io.opentelemetry.sdk.metrics.SdkMeterProvider;
import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
import io.opentelemetry.sdk.resources.Resource;
import io.opentelemetry.sdk.trace.SdkTracerProvider;
import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
import io.opentelemetry.semconv.resource.attributes.ResourceAttributes;
```

Not putting the dependencies and imports may be a real estate saver but this is a time waster for the dev.
Thanks a lot

**Comment by odupuy** [comment]
Hopefully fixed by #2030

**Comment by cartermp** [comment]
Resolved by https://github.com/open-telemetry/opentelemetry.io/pull/2030 - thanks!
